cli/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ _Released 12/3/2024 (PENDING)_
 
 **Breaking Changes:**
 
+- Removed support for Node.js 16 and Node.js 21. Addresses [#29930](https://github.com/cypress-io/cypress/issues/29930).
+- Prebuilt binaries for Linux are no longer compatible with Linux distributions based on glibc <2.28, for example: Ubuntu 14-18, RHEL 7, CentOS 7, Amazon Linux 2. Addresses [#29601](https://github.com/cypress-io/cypress/issues/29601).
```

Comment:
somewhere we want to note that debian 10 is the target build for `better-sqlite3` so if we bump the docker image base to bookworm we need to be aware to then build `better-sqlite3` against the supported glibc version, which is currently `2.28`.